janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/compat/ES6Compat.java

```diff
@@ -25,9 +34,38 @@
 
     private static final IndexFeatures FEATURES = coreFeatures().setWildcardField(CUSTOM_ALL_FIELD).supportsGeoContains().build();
 
+    /**
+     * Script {@code inline} was renamed to {@code source}.
+     * See https://github.com/elastic/elasticsearch/issues/26823
+     */
+    @Override
+    public ImmutableMap.Builder prepareScript(String inline, List<Map<String, Object>> fields) {
+        Map<String, Object> script = ImmutableMap.of(ES_SOURCE_KEY, inline,
+                ES_PARAMS_KEY, ImmutableMap.of(ES_PARAMS_FIELDS_KEY, fields),
+                ES_LANG_KEY, scriptLang());
+        return ImmutableMap.<String, Object>builder().put(ES_SCRIPT_KEY, script);
+    }
+
+    /**
+     * Script {@code inline} was renamed to {@code source}.
+     * See https://github.com/elastic/elasticsearch/issues/26823
+     */
+    @Override
+    public ImmutableMap.Builder prepareScript(String inline) {
+        Map<String, String> script = ImmutableMap.of(ES_SOURCE_KEY, inline, ES_LANG_KEY, scriptLang());
+        return ImmutableMap.<String, Object>builder().put(ES_SCRIPT_KEY, script);
+    }
+
+    /**
+     * Enable params for last version of ES.
```

Comment:
(nitpick) I think this comment doesn't really add anything that isn't clear from the code so I would just remove it, but if you want to keep it, then I would at least remove the mention of this ES version as _last_ as that changes over time and there is in fact already Elasticsearch 7 so it isn't even true right now.

Reply:
Removed.